From 9a195dc573f4eed130773e6ef67efd5f71693e97 Mon Sep 17 00:00:00 2001
From: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
Date: Sat, 5 Nov 2016 15:33:57 +0100
Subject: [PATCH] configure.ac: remove -Werror

-Werror shouldn't be used in releases, as it can break the build with
newer versions of the compiler or the C library, when new warnings
are introduced.

Signed-off-by: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
[Asaf: Modified for version 4.2.0]
Signed-off-by: Asaf Kahlon <asafka7@gmail.com>
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 6868a99b..9a941545 100644
--- a/configure.ac
+++ b/configure.ac
@@ -952,7 +952,7 @@ AS_IF([test "x$enable_Werror" = "xyes" || test "x$enable_Werror" = "xauto"],
         ])])
      AS_IF([test -n "$CPP"],[AS_IF([$CPP --version 2>&1 | grep 'Free Software Foundation' > /dev/null && test "x$GCC" = "xyes"],
         [AC_MSG_NOTICE([Enabling pedantic errors for GNU CPP preprocessor])
-         CPPFLAGS="$CPPFLAGS -pedantic -Werror -Wall -Wc++-compat"
+         CPPFLAGS="$CPPFLAGS -pedantic -Wall -Wc++-compat"
         ],
         [AC_MSG_NOTICE([Not enabling pedantic errors: preprocessor not supported by this recipe (not GNU CPP)])
          AS_IF([test "x$enable_Werror" = "xyes"], [AC_MSG_ERROR([--enable-Werror=yes was requested and can not be satisfied for CPP: $CPP])])
-- 
2.19.1

